nebula_hw_interfaces/src/nebula_hesai_hw_interfaces/hesai_hw_interface.cpp

```diff
@@ -2924,7 +1154,8 @@ HesaiStatus HesaiHwInterface::CheckAndSetConfig(
 #endif
   auto current_return_mode = nebula::drivers::ReturnModeFromIntHesai(
     hesai_config.return_mode, sensor_configuration->sensor_model);
-  auto wait_time = 100ms; // Avoids spamming the sensor, which leads to failure when configuring it.
+  auto wait_time =
+    100ms;  // Avoids spamming the sensor, which leads to failure when configuring it.
```

Comment:
(pedantic)

Reply:
Addressed in [d56b77e](https://github.com/tier4/nebula/pull/126/commits/d56b77e2adffe2c41f8990cfa4926608cb163fa6)